packages/web-client/app/models/workflow/workflow-card.ts

```diff
@@ -58,6 +62,10 @@ export class WorkflowCard extends WorkflowPostable {
       this.workflow?.emit('visible-postables-will-change');
       this.isComplete = true;
     }
+
+    if (this.isComplete && this.cardName) {
+      this.workflow?.session.update('lastCompletedCardName', this.cardName);
+    }
```

Comment:
This wasn't behaving properly when the `check` call above it succeeded asynchronously. I have pushed a commit for this PR to address this.

Reply:
Great, thanks!!